goose3/extractors/schema.py

```diff
@@ -35,10 +35,10 @@ def extract(self):
                 content = json.loads(meta.text_content())
                 if isinstance(content, list):
                     for context in content:
-                        if context["@context"] == "http://schema.org" and context["@type"] in KNOWN_SCHEMA_TYPES:
```

Comment:
perhaps using an in expression would allow for processing of older HTML files:

``` python
if context["@context"] in ("https://schema.org", "http://schema.org") and context["@type"] in KNOWN_SCHEMA_TYPES:
    continue
```

I wonder if that would help resolve the failing tests.

Reply:
> I think the checking both versions either what you are proposing or an `in` expression would allow for older HTML files to continue to work. I wonder if this is why the tests are failing.

Yes, you're right. For the `test_images.py` there was a schema in test data still with http.
I didn't changed it, so there's a way to test both https and https. Let me know if this is ok or if you rather change the test data as well (it's inside `test_reportagenewsarticle.html`).